Controller/CRUDController.php

```diff
@@ -1129,13 +1121,29 @@ protected function redirectTo($object)
         }
 
         if (!$url) {
-            $url = $this->admin->generateUrl('list');
+            return $this->redirectToList();
         }
 
         return new RedirectResponse($url);
     }
 
     /**
+     * Redirect the user to the list view.
```

Comment:
Redirect**s**.

The method redirect**s**. :wink:

Reply:
Copy & paste typo ;)

Reply:
Could be imperative, in which case, no s :P

Reply:
Yeah but we don't use imperative in SymfonyLand. :stuck_out_tongue: